fiovb/host/main.c

```diff
@@ -115,6 +116,43 @@ static int write_persistent_value(const char *name, const char *value)
 	return -1;
 }
 
+static int delete_persistent_value(const char *name)
+{
+	char req1[MAX_BUFFER] = { '\0' };
+
+	struct fiovb_ctx ctx;
+	TEEC_Operation op;
+	TEEC_Result res;
+	uint32_t origin;
+
+	if (strlen(name) >= sizeof(req1)) {
+		fprintf(stderr, "Name value too large (should be up to %d)\n",
+				sizeof(req1));
+		return -1;
```

Comment:
maybe better use POSIX error code ( `-EINVAL;`  as a suggestion) define instead of integer literal?

Reply:
Yes, will change and propose another pr to fix the return code for the other functions.

Reply:
Done.